homeassistant/components/homematicip_cloud/climate.py

```diff
@@ -138,6 +139,22 @@ def hvac_modes(self):
             else [HVAC_MODE_AUTO, HVAC_MODE_COOL]
         )
 
+    @property
+    def hvac_action(self) -> Optional[str]:
+        """
+        Return the current hvac_action.
+
+        This is only relevant for radiator thermostats.
+        """
+        if (
+            self._has_radiator_thermostat
+            and self._device.valvePosition
+            and self._heat_mode_enabled
+        ):
+            return CURRENT_HVAC_HEAT
+
+        return None
```

Comment:
Should this be `CURRENT_HVAC_IDLE` ?

Reply:
I don't think so. The HeatingGroup could contain multiple sources for heating (floor, radiator) at the same time, but we only know that the radiator is currently active as an additional heat source. But we cannot say the HeatingGroup is idle when the radiator is off. We do not have the info if the floor heater is currently heating.
I hope that is understandable.

Reply:
Sorry, i had to rebase and push -f.
I switched the implementation to be for radiators only. In this case HEAT and IDLE action are possible.